KsApi/mutations/inputs/ApplePayParamsTests.swift

```diff
@@ -0,0 +1,20 @@
+@testable import KsApi
+import XCTest
+
+final class ApplePayParamsTests: XCTestCase {
+  func test() {
```

Comment:
`testApplePayParams`? Or something more descriptive 😅

Reply:
Figured seeing as it was one test, but happy to!